payload-schemas/schemas/integration_installation_repositories/added.schema.json

```diff
@@ -138,7 +138,22 @@
         "additionalProperties": false
       }
     },
-    "repositories_removed": { "type": "array", "items": {} },
+    "repositories_removed": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["id", "node_id", "name", "full_name", "private"],
+        "properties": {
+          "id": { "type": "integer" },
+          "node_id": { "type": "string" },
+          "name": { "type": "string" },
+          "full_name": { "type": "string" },
+          "private": { "type": "boolean" }
+        },
+        "additionalProperties": false
+      },
+      "maxItems": 0
+    },
```

Comment:
In theory here, this would the it's proper type, however as this is the added action schema, this array should be empty, which is why `maxItems` is set to 0